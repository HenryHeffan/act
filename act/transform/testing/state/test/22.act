defproc foo (bool a, b, c)
{
  bool d;
  
  prs {
   d => a-
  }
}

foo f;
