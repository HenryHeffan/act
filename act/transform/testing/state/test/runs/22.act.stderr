In expanding foo<> (22.act:1)
WARNING: Process `foo<>': local variable `d': no driver
